Code/Integrator_Project/src/InputCapture.c

```diff
@@ -1,45 +1,49 @@
 /*
- * @file
+ * @file InputCapture.c
  * @brief
  *
  * Copyright DSE - Confidential - All rights reserved
  */
 
 #include "InputCapture.h"
 
-#define CHANNEL_ZERO (0U)
-#define TIMER_ONE_SECOND (1000000000U)
-#define BIT_32 (0x100000000U)
-
-input_capture_info_t input_capture_info;
-uint64_t captureOverflow = 0;
-uint32_t captureCounter = 0;
-uint32_t timeCapturedTruncated = 0;
-uint32_t frequency = 0;
+void InitializeVariablesInputCapture()
+{
+    rpm = ZeroInputCapture;
+    captureOverflow = ZeroInputCapture;
+    captureCounter = ZeroInputCapture;
+    timeCapturedTruncated = ZeroInputCapture;
+    frequency = ZeroInputCapture;
+}
 
 void InitializeInputCapture(void)
 {
     g_input_capture.p_api->open(g_input_capture.p_ctrl, g_input_capture.p_cfg);
+
+    InitializeVariablesInputCapture();
 }
 
 void InputCapture_Callback(input_capture_callback_args_t *p_args)
 {
-    uint32_t pclkdFreq = 0;
-    uint64_t timeCaptured = 0;
+    uint32_t pclkdFreq = ZeroInputCapture;
+    uint64_t timeCaptured = ZeroInputCapture;
 
-    if(CHANNEL_ZERO == p_args->channel)
+    if(CHANNEL_TWO == p_args->channel)
```

Comment:
Why have you changed channel zero to two?

Reply:
Because the channel 0 pin is now used by the **Display**